python/private/pypi/extension.bzl

```diff
@@ -72,11 +72,25 @@ def _platforms(*, python_version, minor_mapping, config):
         version = python_version,
         minor_mapping = minor_mapping,
     )
-    abi = "cp3{}".format(python_version[2:])
 
     for platform, values in config.platforms.items():
+        implementation = values.env["implementation_name"][:2].lower()
```

Comment:
I think this is the spec of how to transform implementation_name to the abi tag prefix: https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/#python-tag

I was going to suggest `env["abi"]`, but it doesn't look like abi is part of the environment marker language.

Reply:
Ah yes, I did it correctly elsewhere. This is a good reminder to move the logic to here.